rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml

```diff
@@ -0,0 +1,36 @@
+title: WScript or CScript Dropper
+id: 002bdb95-0cf1-46a6-9e08-d38c128a6127
+status: experimental
+description: Detects a file ending in jse, vbe, js, vba, vbs written by cscript.exe or wscript.exe
+related: 
+    - id: cea72823-df4d-4567-950c-0b579eaf0846
+      type: derived
+references:
+    - WScript or CScript Dropper (cea72823-df4d-4567-950c-0b579eaf0846)
+author: Tim Shelton
+date: 2022/01/10
+modified: 2022/01/10
```

Comment:
HI no need to add `modified` it is the first release (can not be modified by someone)

Reply:
Reckon it's confusing for new people when updating it, to also remember to add a modified field, vis it already being present.  I think this is 6 one way and half a dozen another

Reply:
modified has been left in, since my changes span 2 days, hope thats ok.